src/serac/physics/heat_transfer.hpp

```diff
@@ -346,7 +343,8 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
     } else {
       // Step the time integrator
       // Note that the ODE solver handles the essential boundary condition application itself
-      ode_.Step(temperature_, time_, dt);
+      double time_tmp = time_;
+      ode_.Step(temperature_, time_tmp, dt);
```

Comment:
why do you make a copy of `time_` before passing to `ode_`?

Reply:
time_ is already passed to the constructor of the time integrator.  It is already incremented there.  This value would also be incremented.  I want to make sure the incrementing wasnt happening twice.

Reply:
Can you put a comment of why you are doing this here and below?

Reply:
I added words.